net.nmlgc.rec98.sh01.yaml

```diff
@@ -0,0 +1,111 @@
+id: net.nmlgc.rec98.sh01
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm19
+build-options:
+  append-path: /usr/lib/sdk/llvm19/bin
+  prepend-ld-library-path: /usr/lib/sdk/llvm19/lib
+finish-args:
+  - --device=dri
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=ipc
+cleanup:
+  - /bin/pango*
```

Comment:
better to make this module specific otherwise it may match things installed by other modules.